src/Education/Commands/RuleHelpUserControl.xaml

```diff
@@ -233,6 +233,12 @@
                 <Setter Property="FontFamily" Value="Courier New" />
                 <Setter Property="FontSize" Value="{DynamicResource {x:Static vsshell:VsFonts.Environment90PercentFontSizeKey}}"/>
             </Style>
+            <Style x:Key="NonCompliant_Diff" TargetType="Span">
```

Comment:
I can't test out the colors yet so I just chose these. Good change they have to be changed but we will have to see. (Added a ticket as a reminder)